### PR TITLE
import script for iconLibrary

### PR DESCRIPTION
**Pull request by pulkitSoni526**
import script for iconLibrary

### CODE REVIEW

File: tools/importer/data.json
Comment (aishwaryamathuria) [inline tools/importer/data.json:1]: Please remove the data file from import script

Comment (aishwaryamathuria) [inline tools/importer/data.json:1]: Please remove the data file from import script

**Review by aishwaryamathuria — CHANGES_REQUESTED** [review]

File: tools/importer/import.js [hunk 1]
```diff
@@ -115,7 +117,7 @@ export default {
       prevOffset = currentOffset;
       offsetDiff = block.querySelectorAll('div').length + 1;
     });
-
+
```
Comment (Deva309) [inline tools/importer/import.js]: Remove these extra spaces added.
Not needed and format the code before raising the PR.

File: tools/importer/import.js [hunk 2]
```diff
@@ -163,7 +165,9 @@ export default {
     const createBlocks = (blockName, divOffset) => {
       const offsetDiff = findOffsetDiff();
       const block = body.querySelectorAll('div')[divOffset + offsetDiff];
+
```
Comment (Deva309) [inline tools/importer/import.js]: Same here

File: tools/importer/import.js [hunk 3]
```diff
@@ -163,7 +165,9 @@ export default {
     const createBlocks = (blockName, divOffset) => {
       const offsetDiff = findOffsetDiff();
       const block = body.querySelectorAll('div')[divOffset + offsetDiff];
+    
       switch (blockName) {
+
```
Comment (Deva309) [inline tools/importer/import.js]: same here

File: tools/importer/import.js [hunk 4]
```diff
@@ -206,13 +210,18 @@ export default {
         case constants.gradientLine:
           createGradientLineBlock(block, document);
           break;
+          case constants.gradientLine:
```
Comment (Deva309) [inline tools/importer/import.js]: Could you please format the code correctly?
Also, run eslint as well.

File: tools/importer/import.js [hunk 5]
```diff
@@ -206,13 +210,18 @@ export default {
...
+            iconblock(block, document);
+            break;
+            case constants.iconLibrary:
+              iconLibrary(block, document);
+            break;
         default:
-          // default
+          // defaulticonblock
```
Comment (Deva309) [inline tools/importer/import.js]: why defaulticonblock?

File: tools/importer/rules/iconLibrary.js [hunk 6]
```diff
@@ -0,0 +1,39 @@
+export default function iconLibrary(block, document) {
```
Comment (Deva309) [inline tools/importer/rules/iconLibrary.js]: Format this file and run eslint as well

File: tools/importer/services/getBlocks.js [hunk 7]
```diff
@@ -3,7 +3,8 @@ export default async function getBlocks(url) {
     const parsedURL = new URL(url);
     const relativePath = parsedURL.pathname.slice(1).split('/');
     relativePath.pop();
-    const jsonPath = ['https://aishwaryamathuria.github.io/franklin-cc-json/cc', relativePath.join('/'), 'info.json'].join('/');
+    // const jsonPath = ['https://aishwaryamathuria.github.io/franklin-cc-json/cc', relativePath.join('/'), 'info.json'].join('/');
```
Comment (Deva309) [inline tools/importer/services/getBlocks.js]: Remove these changes if you have used it at your local only

**Review by Deva309 — CHANGES_REQUESTED** [review]

**Review by Deva309 — APPROVED** [review]

**Pull request merged**